README.md

```diff
@@ -364,5 +364,22 @@ See the `config/stack/overrides` directory for details about where the
 overrides live. See the `stack.Makefile` for details about how
 `local-build` works.
 
+### Setup RBAC
+
+We can setup extra permissions to grant access to the resources which are not part of our stack. This can be done by specifing the permissions in the CustomResourceDefinition YAML of our custom resource.
```

Comment:
> This can be done by specifing the permissions in the CustomResourceDefinition YAML of our custom resource.

This seems like it is not entirely accurate. It sounds like this is talking about specifying permissions in the CRD yaml, but that isn't how it works for a stack. We specify the permissions in the stack definition (the content shown below in the example is a stack definition, which is separate from a CRD). One way to word this would be something like:

```
This can be done by specifying the permissions in the stack definition YAML.
```

Reply:
You are right. I mistook it for a CRD. Thanks for pointing it out!

Reply:
Fixed